package.json

```
@@ -23,7 +23,8 @@
     "test:watch": "vitest",
     "test:size": "npm i && size-limit",
     "test:coverage": "vitest run --coverage",
-    "all": "npm run lint && npm run check && npm t && npm run build && npm run preview"
+    "all": "npm run lint && npm run check && npm t && npm run build",
+    "host": "npm run lint && npm run check && npm t && npm run build && npm run preview"
```

Since `host` extends `all`, I would write it like,
```suggestion
    "host": "npm run all && npm run preview"
```